### PR TITLE
Rank times standardisation, field commander max rank time fix

### PR DESCRIPTION
**Pull request by EF25**
## About The Pull Request

Changes the time of max rank for the Field Commander, allows it to extend out to infinity, as it should.
Standardises rank times, better distribution for some roles that previously had a very cluttered lineup.

Adds comments in code next to times in hours, bit of qol.

## Why It's Good For The Game

Rank times were a mess, now all are standardised to set times depending on position, e.g. all of command having 0/25/125.
Made on suggestions by RenGusta.

Some roles like the PO had a very compressed set of ranks that got maxed by hour 50, fixed by extending times for it and roles with a similar problem.

## Changelog
:cl: E25
tweak: Marine ranks are now 0/10/100/300/1000 hours.
tweak: All Command ranks are now 0/25/125 hours.
tweak: PO, RO and Ship engie ranks are now 0/25/50/100 hours.
fix: Fixed Field Commander's max rank not extending out to infinity.
/:cl:

### CODE REVIEW

File: code/datums/jobs/job/shipside.dm [hunk 1]
```diff
@@ -67,11 +67,11 @@ Godspeed, captain! And remember, you are not above the law."})
...
 		return
 	switch(playtime_mins)
-		if(0 to 500)
+		if(0 to 1500) // starting
 			new_human.wear_id.paygrade = "O6"
-		if(501 to 10000)
+		if(1501 to 7500) // 25hrs
 			new_human.wear_id.paygrade = "O7"
-		if(10001 to INFINITY)
+		if(7501 to INFINITY) //125 hrs
```
Comment (TiviPlus) [inline code/datums/jobs/job/shipside.dm:74]: This is pretty big jumps are you sure about this

Comment (EF25) [reply code/datums/jobs/job/shipside.dm:74]: All command roles meant to have the max rank at 125 hrs, it's only being reduced by about 42hrs, which isn't that lot when taking the overall hours into account.
If someone spends over 125hrs playing Cap I doubt that an extra 40-ish hrs would make much of a difference.

Comment (TiviPlus) [reply code/datums/jobs/job/shipside.dm:74]: Mhmm sure I guess

**Review by TiviPlus — APPROVED** [review]
Fix conflicts and this is good to go

**Review by TiviPlus — CHANGES_REQUESTED** [review]
And the Changelog.html too

**Comment by EF25** [comment]
Changelog got in again...
No idea how...

**Pull request closed without merging**